src/gate_detect/src/Interpolator.cpp

```diff
@@ -0,0 +1,45 @@
+//
+// Created by cam on 09/03/19.
+//
```

Comment:
Would be helpful to replace this with a more descriptive comment

Reply:
Done! My bad